src/algebra/monoid_algebra/basic.lean

```diff
@@ -694,6 +694,16 @@ lemma lift_unique (F : monoid_algebra k G →ₐ[k] A) (f : monoid_algebra k G)
   F f = f.sum (λ a b, b • F (single a 1)) :=
 by conv_lhs { rw lift_unique' F, simp [lift_apply] }
 
+/-- If `f : G → H` is a homomorphism between two magmas, then
```

Comment:
There'll be a corresponding version for additive magmas / `add_monoid_algebra` too.

Reply:
:facepalm:

Reply:
Additive added.  The doc-string feels a little weird "add magma algebras"?  I went on auto-pilot, as if I were a working version of `to_additive`, but feel free to suggest a different name.

Reply:
I think "additive magma algebras" is slightly better

Reply:
Doc-string changed!

Reply:
I'll be *very* quick to type `bors r+` once CI finishes!